docs/csharp/language-reference/compiler-messages/cs1705.md

```diff
@@ -161,7 +161,7 @@ class Tester
   
      `csc /reference:C:\\CS1705.dll /reference:CS1705.dll /reference:CS1705c.dll CS1705d.cs`  
   
-     Although the program compiles when you use this command, it still does not run. To enable the program to run, you can provide an application configuration file that includes a [\<dependentAssembly> element](http://msdn.microsoft.com/library/14e95627-dd79-4b82-ac85-e682aa3a31d8) that uses [\<assemblyIdentity>](http://msdn.microsoft.com/library/cea4d187-6398-4da4-af09-c1abc6a349c1) and [\<codeBase>](http://msdn.microsoft.com/library/d48a3983-2297-43ff-a14d-1f29d3995822) child elements to specify the location of version 1.0 of the DLL. For more information about configuration files, see [Configuring Apps](http://msdn.microsoft.com/library/86bd26d3-737e-4484-9782-19b17f34cd1f).  
+     Although the program compiles when you use this command, it still does not run. To enable the program to run, you can provide an application configuration file that includes a [\<dependentAssembly> element](http://msdn.microsoft.com/library/14e95627-dd79-4b82-ac85-e682aa3a31d8) that uses [\<assemblyIdentity>](http://msdn.microsoft.com/library/cea4d187-6398-4da4-af09-c1abc6a349c1) and [\<codeBase>](http://msdn.microsoft.com/library/d48a3983-2297-43ff-a14d-1f29d3995822) child elements to specify the location of version 1.0 of the DLL. For more information about configuration files, see [Configuring Apps](../../../framework/configure-apps/index.md).  
```

Comment:
shouldn't links such as https://msdn.microsoft.com/library/14e95627-dd79-4b82-ac85-e682aa3a31d8 be changed to https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/runtime/dependentassembly-element.md?